### ISSUE TITLE
World processor: ability to remove ghosts

### ISSUE THREAD

**Issue by Sector171** [Anuken/Mindustry-Suggestions#4344]
### Describe the content or mechanics you are proposing.

The ability using wp to remove destroyed buildings ghosts RemoveGhost(team,x1,y1,x2,y2) from specific area.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

These ghosts are a big obstacle in making non ordinary maps. We really need this feature asap.

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by Darkness6030** [comment]
@Sector171 rules -> ghostBlocks -> disable it

**Comment by Sector171** [comment]
@Darkness6030 

> rules -> ghostBlocks -> disable it

Who told you I want to disable it, I don't want to disable it, I want to have the ability to clear the area when I want that, it is important. I want build tower to work of course.
Also I checked the rules in my mobile didn't find anything about ghostblocks!! There is no option like that, also I checked setrule nothing about it. Even though it will not solve the problem.

Anyway, it will be good solution to add it to setrule, setrule-->ghostblocks--> (false,true) but in a dynamic way(can be changed during the game), also when set to false this will clean all the ghost for all teams in the map, yeah that's fine and that's what I want.
